src/twisted/web/http.py

```diff
@@ -1211,7 +1210,7 @@ def write(self, data):
                 and self.method != b"HEAD"
                 and self.code not in NO_BODY_CODES
             ):
-                headers.append((b"Transfer-Encoding", b"chunked"))
+                self.responseHeaders.setRawHeaders("Transfer-Encoding", [b"chunked"])
```

Comment:
Maybe add a comment, that we overwrite any previous encoding that might have been explicitly set by the resource.

Reply:
Previously we would just have... both? It's not something users should set, either way.